File: Alhambra_Android/app/src/main/java/com/alhambra/network/PackedJSONMessages.java
package com.alhambra.network;

import java.util.ArrayList;

public class PackedJSONMessages {
    public static final String ACTION_NAME="PackedMessage";
    public ArrayList<String> actions = new ArrayList<String>();

    public void addAction(String actionName, Object data){
        actions.add(JSONUtils.createActionJson(actionName,data));
    }

    public void addString(String json){
        actions.add(json);
    }
}

File: AlhambraXR/Assets/Scripts/Network/PackedJSONMessages.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using UnityEngine;

// if you care about sequence of message, use this!
[Serializable]
public class PackedJSONMessages
{
    public const string ACTION_NAME = "PackedMessage";
    [SerializeField]
    public List<string> actions = new List<string>();

    public override string ToString()
    {
        return JSONMessage.ActionJSON(ACTION_NAME, this);
    }

    public void AddAction<T>(string actionName, T obj)
    {
        actions.Add(JSONMessage.ActionJSON(actionName, obj));
    }

    public void AddString(string json)
    {
        actions.Add(json);
    }

    
}
